src/core/ee/vu.cpp

```diff
@@ -465,10 +465,16 @@ void VectorUnit::rsqrt(uint8_t ftf, uint8_t fsf, uint8_t reg1, uint8_t reg2)
     if (!denom)
     {
         printf("[VU] RSQRT by zero!\n");
-        exit(1);
+        if ((gpr[reg1].u[fsf] & 0x80000000) != (gpr[reg2].u[ftf] & 0x80000000))
+            Q.u = 0xFF7FFFFF;
+        else
+            Q.u = 0x7F7FFFFF;
+    }
```

Comment:
I'm assuming this should set the status flags? See VectorUnit::div.

Reply:
Yep, my bad, missed that, will update